bench/versions/benchmarks.js

```diff
@@ -47,6 +49,7 @@ register(new PaintStates(center));
 LayerBenchmarks.forEach((Bench) => register(new Bench()));
 register(new Load());
 register(new LayoutDDS());
+register(new SymbolLayout(style, styleLocations.map(location => location.tileID[0])));
```

Comment:
This seems like it would be useful in `styles/benchmarks` too

Reply:
Yeah, it could be useful, but I'm kind of worried that the style benchmarks are already really long running and kind of intimidating to browse through -- I'm wary of adding lots more benchmarks to them. My thinking is kind of that for the style benchmarking we want to be able to catch changes in overall performance for specific areas (given changes in the style), whereas for version benchmarking we want to catch changes in performance of specific parts of the code over a broad set of areas (given changes in the code).

cc @nickidlugash

Reply:
I think that makes sense